src/item.cpp

```diff
@@ -2682,91 +2682,116 @@ void item::final_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         }
 
         if( parts->test( iteminfo_parts::DESCRIPTION_FLAGS_FITS ) ) {
-            if( has_flag( "FIT" ) ) {
-                if( sizing_level == sizing::human_sized_human_char ) {
+            switch( sizing_level ) {
+                case sizing::human_sized_human_char:
+                    if( has_flag( "FIT" ) ) {
+                        info.push_back( iteminfo( "DESCRIPTION",
```

Comment:
You can get some horizontal space by using `emplace_back` instead of `push_back`:
```suggestion
                        info.emplace_back( "DESCRIPTION",
                                           _( "* This clothing <info>fits</info> you "
```
That way you don't need an explicit constructor call and more usable space remains on that line.

Reply:
Oh wow, that is neat. I wasn't aware about emplace_back but from what I'm reading, it can be faster sometimes (due to push_back not being guaranteed to not make unnecessary copies)… Would it be a good idea to make this change for all instances of `info.push_back( item_info(` in item.cpp? (There are 180 total.) And would it be ok in this PR?

Reply:
> Would it be a good idea

Yes.
> And would it be ok in this PR?

Probably better to make a separate one.